apps/web/src/components/layouts/AppLayout/Header/Rewards/Content/RewardItem.tsx

```diff
@@ -0,0 +1,57 @@
+'use client'
+
+import { useMemo } from 'react'
+
+import { REWARD_VALUES, RewardType } from '@latitude-data/core/browser'
+import { Button, cn, Icon, Text } from '@latitude-data/web-ui'
+import useRewards from '$/stores/rewards'
+
+export function RewardItem({
+  description,
+  type,
+  onClick,
+}: {
+  description: string
+  type: RewardType
+  onClick: () => void
+}) {
+  const { data: claimedRewards, isLoading } = useRewards()
+
+  const isClaimed = useMemo(() => {
+    if (isLoading || !claimedRewards) return false
+    return claimedRewards.some((r) => r.reward_type === type)
```

Comment:
Why this column is with underscore `reward_type`. We use `rewardType`

Reply:
true!